ports/mimxrt/boards/MIMXRT1010_EVK/mpconfigboard.h

```diff
@@ -23,6 +23,12 @@
     { IOMUXC_GPIO_08_LPUART3_TXD }, { IOMUXC_GPIO_AD_07_LPUART3_RXD }, \
     { IOMUXC_GPIO_AD_02_LPUART4_TXD }, { IOMUXC_GPIO_AD_01_LPUART4_RXD },
 
+#define IOMUX_TABLE_UART_CTS_RTS \
+    { IOMUXC_GPIO_08_LPUART1_CTS_B }, { IOMUXC_GPIO_07_LPUART1_RTS_B }, \
+    { 0 }, { 0 }, \
+    { 0 }, { 0 }, \
```

Comment:
What happens if this third uart is used with flow control enabled, will it detect that flow is not supported on this uart and raise a Python exception?

Reply:
Yes it should raise an exception if RTS or CTS mux register is zero.